docs/source/develop.rst

```diff
@@ -22,6 +22,26 @@ and install it from source::
    cd distributed
    python setup.py install
 
+Using conda, for example::
+
+   git clone git@github.com:{your-fork}/distributed.git
+   cd distributed
+   conda create -y -n distributed python=3.6
+   conda activate distributed
+   pip install -U -r requirements.txt
+   pip install -U -r dev-requirements.txt
+   pip install -e .
+
+To keep a fork in sync with the upstream source::
```

Comment:
In the Dask developer docs we link to Pandas' documentation on using `git` which is quite thorough. I'd prefer to leave this section out in favor of users going to the relevant section in the Pandas docs

Reply:
Whie it's easier to keep things DRY from a documentation perspective, extra links does add a burden on the user to skip around and sift through the volumes of docs about pandas development to find the kernels that are needed for dask.  A few lines for copy-paste that can get it up and running in a few seconds could be useful, but not a substitute for the other docs.

Reply:
I'll split this and the other snippet out of this PR.